src/rgw/rgw_log.cc

```diff
@@ -211,8 +211,12 @@ static void log_usage(struct req_state *s, const string& op_name)
   string p = payer.to_str();
   rgw_usage_log_entry entry(u, p, bucket_name);
 
-  uint64_t bytes_sent = s->cio->get_bytes_sent();
-  uint64_t bytes_received = s->cio->get_bytes_received();
+  uint64_t bytes_sent = 0;
+  uint64_t bytes_received = 0;
+  if (ACCOUNTING_IO(s)) {
+    bytes_sent = ACCOUNTING_IO(s)->get_bytes_sent();
+    bytes_received = ACCOUNTING_IO(s)->get_bytes_received();
+  }
```

Comment:
consider storing the result of `ACCOUNTING_IO(s)` in a temporary so we only `dynamic_cast` once